engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -478,26 +478,30 @@ class WorkflowActor(workflowToStart: WorkflowToStart,
     goto(finalState) using data.copy(currentLifecycleStateActor = None)
   }
 
-  private[workflow] def makeFinalizationActor(workflowDescriptor: EngineWorkflowDescriptor, jobExecutionMap: JobExecutionMap, workflowOutputs: CallOutputs) = {
-    val copyWorkflowOutputsActorProps = stateName match {
-      case InitializingWorkflowState => None
-      case _ => Option(CopyWorkflowOutputsActor.props(workflowIdForLogging, ioActor, workflowDescriptor, workflowOutputs, stateData.initializationData))
+  private[workflow] def makeFinalizationActor(workflowDescriptor: EngineWorkflowDescriptor, jobExecutionMap: JobExecutionMap, workflowOutputs: CallOutputs): ActorRef = {
+    val actorsPropsOption: (Option[Props], Option[Props]) = stateName match {
```

Comment:
I still think that it would be more appropriate to introduce a case class here and call it something like `FinalizationActorsProps`. It would add more context on this object and e.g. `finalizationActorsProps.copyWorkflowOutputsActorProps` on a line `494` would be way more clear and understandable.